python/pylibcugraph/pylibcugraph/force_atlas2.pyx

```diff
@@ -196,14 +205,16 @@ def force_atlas2(ResourceHandle resource_handle,
     ...     resource_handle, graph_props, srcs, dsts, weight_array=weights,
     ...     store_transposed=False, renumber=False, do_expensive_check=False)
     >>> (vertices, x_axis, y_axis) = pylibcugraph.force_atlas2(
-    ...     resource_handle, None, G, 500, None, None, None, True, False, False, None,
-    ...     1.0, 1.0, True, 0.5, 2.0, False, None, 1.0, False, False)
+    ...     resource_handle, 42, G, 500, None, None, None, True, False, False,
+    ...     None, None, 100.0, 1.0, 1.0, False, 0.5, 2.0, False, 1.0, None, None,
+    ...     None, None, False, False)
     >>> vertices
     [   0  1   2   3   4   5    ]
     >>> x_axis
-    [ 5.444471    0.4794112   1.2495936  -0.01039529 -1.1892298  -1.5889403 ]
+    [-7.7015705   7.6763854  -2.7651896  -0.02446137  1.6971487  -1.2822613 ]
     >>> y_axis
-    [-1.4304754e+01 -3.8182523e+00  3.8365445e+00  8.3183739e-03 -8.3009762e-01 -1.9155006e-01
+    [ 23.276543     7.9067745   13.618961    -0.07172047  -6.8321953  -11.90544   ]
+
```

Comment:
Thanks for remembering the docstring example.

I'm assuming these values are drastically different because the call in the example is also different/updated (and not simply because support for `mass` was added)?